integration-tests/helpers.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+const express = require('express')
+const bodyParser = require('body-parser')
+const msgpack = require('msgpack-lite')
+const codec = msgpack.createCodec({ int64: true })
+const EventEmitter = require('events')
+const { fork } = require('child_process')
+const http = require('http')
+
+class FakeAgent extends EventEmitter {
+  constructor (port = 0) {
+    super()
+    const agent = express()
+    agent.use(bodyParser.raw({ limit: Infinity, type: 'application/msgpack' }))
+    agent.use((req, res) => {
```

Comment:
This should only work on the correct route.

Reply:
Good call! Will fix.